mobile/library/swift/EngineBuilder.swift

```diff
@@ -726,6 +756,114 @@ open class EngineBuilder: NSObject {
   }
 
   func bootstrapDebugDescription() -> String {
-    self.makeConfig().bootstrapDebugDescription()
+    let objcDescription = self.makeConfig().bootstrapDebugDescription()
+#if canImport(EnvoyCxxSwiftInterop)
+    assert(
+      self.generateBootstrap().debugDescription == objcDescription,
+      "Swift bootstrap is different from ObjC bootstrap"
+    )
+#endif
+    return objcDescription
   }
 }
+
+#if canImport(EnvoyCxxSwiftInterop)
+private extension EngineBuilder {
+  func generateBootstrap() -> Bootstrap {
+    var cxxBuilder = Envoy.Platform.EngineBuilder()
+    cxxBuilder.addLogLevel(self.logLevel.toCXX())
```

Comment:
just pointing out that we are not doing this in obj-c builder but I think that it's fine. I think that it will be effectively unused as we take logLevel from outside of the bootstrap https://github.com/envoyproxy/envoy/blob/4e6956f77ce9685237f9a523e0aa59ef5f171455/mobile/library/objective-c/EnvoyEngineImpl.mm#L529